CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Significant changes to public interface: ImmutableX
```

Comment:
This line can probably be removed now

Reply:
[627d900](https://github.com/immutable/imx-core-sdk/pull/185/commits/627d9004b0fdcff0cad0ca82adc9e0a9bf8bbb66)